client/src/QuestionsTab.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I really like your components' naming conventions. It is clear and easy to understand. 💯